configure.ac

```diff
@@ -57,30 +85,88 @@ AC_ARG_WITH([state-file-path],
         STATE_FILE_PATH="$withval"
         ;;
     esac
-  ],
-  [
-    STATE_FILE_PATH="/var/lib/logrotate.status"
   ])
 AC_DEFINE_UNQUOTED([STATEFILE], "$STATE_FILE_PATH")
 AC_SUBST(STATE_FILE_PATH)
 
 AC_ARG_WITH([default-mail-command],
   AC_HELP_STRING([--with-default-mail-command=COMMAND],
-                 [default mail command (based on config.h if not given)]),
+                 [default mail command (default: /bin/mail -s)]),
```

Comment:
Should we remove the ```-s``` then?

Because ```-s``` is used by default for ```*-netbsd*``` only if I understand it correctly.